Sources/Todo.WebApi/Models/TodoItemQueryModel.cs

```diff
@@ -7,5 +9,15 @@ public class TodoItemQueryModel
         public string NamePattern { get; set; }
 
         public bool? IsComplete { get; set; }
+
+        [Range(1, 1000)] 
+        public int PageSize { get; set; } = 25;
+
+        [Range(0, int.MaxValue)] 
+        public int PageIndex { get; set; } = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this initialization to 'PageIndex', the compiler will do that for you.](https://app.codacy.com/manual/satrapu/aspnet-core-logging/pullRequest?prid=5148460)